src/StackExchange.Utils.Http/HttpSettings.cs

```diff
@@ -78,5 +85,15 @@ public HttpSettings()
             // Default pool by default
             ClientPool = new DefaultHttpClientPool(this);
         }
+
+        private const string Switch_AllowUnencryptedHttp2 = "System.Net.Http.SocketsHttpHandler.Http2UnencryptedSupport";
+        /// <summary>
+        /// Allows HttpClient to use HTTP/2 without TLS
```

Comment:
Let's note this is *app global* and affects more than this library? I'd prefer that be in both the description and property name to be super clear - remember how confusing `ServicePointManager` is to people...